src/connectors/bullofheaven.com.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const artistName = 'Bull of Heaven';
+
+const trackRegex = /\d+ - /;
+
+const playButtonSelector = '.jp-play';
+
+Connector.playerSelector = '#jp_container_N';
+
+Connector.getArtist = () => artistName;
+
+Connector.getTrack = () => {
+	const trackElement = document.querySelector('.jp-playlist-current .playlist-title');
+	return trackElement.textContent.replace(trackRegex, '');
+};
+
+Connector.trackArtSelector = '#jp_poster_0';
+
+Connector.currentTimeSelector = '.jp-current-time';
+
+Connector.durationSelector = '.jp-duration';
+
+Connector.isPlaying = () => !Util.isElementVisible(playButtonSelector);
```

Comment:
Why `Connector.playButtonSelector` is not used here?

Reply:
In this case it's the negative of what's expected. e.g. `playButtonSelector` is expecting it to be visible - in this case it's the reverse.

Reply:
When a track is play, the play button is expected to be hidden. The default implemetation perform the same check:
https://github.com/web-scrobbler/web-scrobbler/blob/a97bae9740438708ae85b77b8acc47940eeeaf68/src/core/content/connector.js#L333-L338

Reply:
Weird - tried this before and the logic seemed reverse. Now it's working :boom: